src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {person.age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
You've already destructured the person object
```suggestion
      {age && <p className="Person__age">{`I am ${age}`}</p>}
```